ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingConfig.java

```diff
@@ -239,7 +251,8 @@ public boolean equals(Object o) {
   @Override
   public int hashCode() {
     return Objects.hashCode(enabled, targetQuota, optimizerGroup, maxExecuteRetryCount, maxCommitRetryCount, targetSize,
-        maxFileCount, openFileCost, fragmentRatio, minorLeastFileCount, minorLeastInterval, majorDuplicateRatio,
+        maxTaskSize, maxFileCount, openFileCost, fragmentRatio, minorLeastFileCount, minorLeastInterval,
+        majorDuplicateRatio,
```

Comment:
Leaving the single paramter in one line here makes the format quite strange.

Reply:
OK, I will modify it. Code refactoring brought about this strange state.